src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/ActivitySource.cs

```diff
@@ -9,9 +9,10 @@ namespace System.Diagnostics
 {
     public sealed class ActivitySource : IDisposable
     {
+        private SynchronizedList<ActivityListener>? _listeners;
```

Comment:
Nit: I'm curious why you moved this up?  We don't have any strict requirements around this, but _generally_ we have static fields before instance fields, as it was previously written.

Reply:
No real reason more than it was part of previous changes which I reverted. I moved it back now to its original place.